### PR TITLE
Change url in setup.py

### PR DESCRIPTION
**Pull request by fengzhou-msft**
Use a URL to the extension directory in extension's `setup.py`. It will eventually be in package metadata and `index.json`,  and expose to users through doc or `az extension show`.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

File: src/portal/setup.py [hunk 1]
```diff
@@ -47,8 +47,7 @@
     # TODO: Update author and email, if applicable
     author='Microsoft Corporation',
     author_email='azpycli@microsoft.com',
-    # TODO: consider pointing directly to your source code instead of the generic repo
-    url='https://github.com/Azure/azure-cli-extensions',
+    url='https://github.com/Azure/azure-cli-extensions/tree/master/src/portal,
```
Comment (fengzhou-msft) [inline src/portal/setup.py:50]:
```suggestion
    url='https://github.com/Azure/azure-cli-extensions/tree/master/src/portal',
```

**Comment by yonzhan** [comment]
setup.py

**Comment by fengzhou-msft** [comment]
> will we have new packages?

I'm not planning to relase new packages just for this change. They will be picked up when next version is released for other features.

I think checking into master branch would at least change the links in this page: https://docs.microsoft.com/en-us/cli/azure/azure-cli-extensions-list?view=azure-cli-latest

**Comment by fengzhou-msft** [comment]
The URL change for aem, eventgrid, hardware-security-modules, log_analytics_solution, privatedns will be submitted in another PR as their tests fail now.

**Pull request merged**